api_shiftme/app/models/request.rb

```diff
@@ -0,0 +1,4 @@
+class Request < ApplicationRecord
+  belongs_to :requester, class_name: 'User'
+  belongs_to :requested, class_name: 'User'
+end
```

Comment:
Did you forget something?I think that this is missing: 
belongs_to : current_Shift, class_name: 'Shift'
belongs_to : requested_Shift, class_name: 'Shift'